lookout/style/format/rules.py

```diff
@@ -112,7 +113,8 @@ def apply(self, X: numpy.ndarray, return_winner_indices=False
         return prediction
 
     def predict(self, X: numpy.ndarray, vnodes_y: Sequence[VirtualNode],
-                vnodes: Sequence[VirtualNode], language: str, return_originals: bool = False
+                vnodes: Sequence[VirtualNode], language: str, feature_extractor: FeatureExtractor,
```

Comment:
We can take `language` from `feature_extractor`. And actually `language` should be completely abstracted from the code here and in `postprocess` and all the language-dependent functions should be consumed from `feature_extractor`